components/RandomQueue.js

```diff
@@ -5,10 +5,18 @@ import { ListGroup } from 'reactstrap'
 import Question from './Question'
 
 const STAFF = [
-  'Nathan Walters',
-  'Genna Helsel',
-  'Jordi Paris Ferrer',
-  'Wade Fagen-Ulmschneider',
+  'TA - Nathan Walters',
```

Comment:
Not a huge fan of TA/CA prefix - looks a bit cluttered and isn't really reflective of how the Queue actually looks.

Reply:
:+1: